### PR TITLE
Add systemd files

### PR DESCRIPTION
**Pull request by madonius**
@problame the files are meant for the usage in our VM. Please have a look and feel free to correct to your liking

### CODE REVIEW

**Comment by problame** [comment]
Despite the service already running unprivileged, please add cheap systemd isolation features compatible with Debian Stretch:

* http://0pointer.de/blog/projects/security.html

Here's what I use for zrepl. If you're motivated, try a syscall whitelist :)

```
[Unit]
Description=zrepl daemon
Documentation=https://zrepl.github.io

[Service]
Type=simple
ExecStart=/usr/local/bin/zrepl --config /etc/zrepl/zrepl.yml daemon
RuntimeDirectory=zrepl
RuntimeDirectoryMode=0700

ProtectSystem=strict
#PrivateDevices=yes # Does not work because we need access to /dev/zfs
ProtectKernelTunables=yes
ProtectControlGroups=yes
PrivateTmp=yes
#PrivateUsers=yes # Did not work on Fedora 28
ProtectKernelModules=true
RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6
RestrictNamespaces=true
RestrictRealtime=yes
SystemCallArchitectures=native

# BEGIN FEDORA 28
ProtectHome=tmpfs
# Syscall blacklist (should be fairly stable)
SystemCallFilter=~ @mount @aio @cpu-emulation @keyring @memlock @module @obsolete @privileged @raw-io @reboot @setuid @swap @sync @timer @debug @clock @chown @resources
# Syscall whitelist (not sure how stable)
#SystemCallFilter=@default @file-system @process @basic-io @ipc @network-io @signal @io-event brk mprotect sched_getaffinity ioctl getrandom
# END FEDORA

# DEBIAN STRETCH
ProtectHome=read-only
SystemCallFilter=~ @mount @cpu-emulation @keyring @module @obsolete @privileged @raw-io @debug @clock @resources
# END DEBIAN

[Install]
WantedBy=multi-user.target


```

File: systemd/ics2entropiawiki.service [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Retrieve ics File and publish it to the entropia wiki
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/ics2entropiawiki -c /etc/ics2entropiawiki/config.ini
+User=ics2entropiawiki
+Group=systemd-journal
```
Comment (problame) [inline systemd/ics2entropiawiki.service]: why?

File: systemd/ics2entropiawiki.service [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Retrieve ics File and publish it to the entropia wiki
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/ics2entropiawiki -c /etc/ics2entropiawiki/config.ini
+User=ics2entropiawiki
```
Comment (problame) [inline systemd/ics2entropiawiki.service]: document the requirement for this user to be present in the README.
Ideally, with a copy-pastable command to add a system user called `ics2entropiawiki`.

File: systemd/ics2entropiawiki.service [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[Unit]
```
Comment (problame) [inline systemd/ics2entropiawiki.service:1]: `[Install]` section missing. -> `multi-user.target`

**Review by problame — CHANGES_REQUESTED** [review]

File: systemd/ics2entropiawiki.service (hunk 2 above)
Comment (madonius) [reply systemd/ics2entropiawiki.service]: I would do that, ideally I would provide something script like doing everything that needs to be done. A `.deb` maybe?

File: systemd/ics2entropiawiki.service (hunk 1 above)
Comment (madonius) [reply systemd/ics2entropiawiki.service]: Bare with me here, I have never written a system-wide systemd service file.
What would you recommend?

**Comment by madonius** [comment]
Thanks for the "template" will fix that tonight

File: systemd/ics2entropiawiki.service (hunk 2 above)
Comment (problame) [reply systemd/ics2entropiawiki.service]: phew, didn't know you wanted to go as far as a `.deb`.

**Review by problame — APPROVED** [review]

**Pull request merged**